nrf_802154/doc/CHANGELOG.rst

```diff
@@ -9,37 +9,6 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
-nRF Connect SDK 1.7.0 - nRF 802.15.4 Radio Driver
```

Comment:
Looks like this change was not backported.

Reply:
backported